creator-node/src/routes/tracks.js

```diff
@@ -102,8 +102,10 @@ async function handleResumableUpload (req, res, next) {
           }
         }
       )
+      req.logger.info(`Added transcode task, uuid=${req.logContext.requestID}`)
```

Comment:
this log is probably not necessary since we have a log in the add fn https://github.com/AudiusProject/audius-protocol/pull/2138/files#diff-57270c24153b3ccb860bbc7f62c0ede8ffd4a4550ddfe43e2163bd7448473dddR70

Reply:
The other one just says we're adding the job, not that we did. But yeah, this log should go within that function and not here. Thanks!